application/comit_node/src/swap_protocols/rfc003/ethereum/contract_templates/ether_contract.asm

```diff
@@ -38,13 +35,10 @@
 
     timestamp
 
-    // Subtract current timestamp from deployment timestamp (pushed to stack as first instruction of this contract)
-    sub
-
-    // Placeholder for relative expiry time
-    0x20000002
+    // Placeholder for refund timestamp
+    0x50000005
```

Comment:
🙃You could leave this as `0x20000002` in order to keep the incremental order of the placeholders